src/internal/sudoloop.rs

```diff
@@ -6,13 +6,16 @@ use crate::ShellCommand;
 /// Loop sudo so longer builds don't time out
 #[allow(clippy::module_name_repetitions)]
 pub fn start_sudoloop() {
-    prompt_sudo();
```

Comment:
This one needs to stay. Otherwise ame will continue and overwrite the sudo prompt in the terminal (since the sudo prompt was created from a different thread).